README.md

```diff
@@ -1542,6 +1542,238 @@ data: {
+
+### Create Payment Channel
+
+Choose a globally unique identifier for `paymentChannel` and create it. Optionally provide a `rawtx` for the initial transaction statte.
+
+Once a payment channel is initialized with the first transaction, then all subsequent updates must use the same inputs and adhere to the rules
+regarding nSequence, nLockTime. See: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2013-April/002417.html
+
+`POST /api/v1/paymentchannel/:paymentChannelId`
+
+```javascript
+{
+  "rawtx": '003222...' // Optional initial rawtx to initialize with. If left empty then use "Updates" to change
+}
+
+```
+
+### Add Transaction to Payment Channel
+
+`POST /api/v1/paymentchannel/:paymentChannelId/tx`
+
+```javascript
+{
+  "rawtx": '003222...' // Optional initial rawtx to initialize with. If left empty then use "Updates" to change
+}
+
+```
+
+Response:
+
+```javascript
+{
+  "status": 200,
+  "errors": [],
+  "result":  {
+    "closed": false,  // Whether the channel is closed manually or one of the inputs was spent by an actual tx broadcasted
+    "state": {
+      "id": 135,
+      "txid": "...",
+      "rawtx": "000022...",
+      "paymentchannelid": "my-id-payment-channel",
+      "updated_at": 1114554332,
+      "created_at": 1114554332
+    }
+  }
+}
+```
+
+### Get Payment Channel State
+
+`GET /api/v1/paymentchannel/:paymentchannelid`
+
+Response:
+
+```javascript
+{
+  "status": 200,
+  "errors": [],
+  "result":
+  {
+    "closed": false,  // Whether the channel is closed manually or one of the inputs was spent by an actual tx broadcasted
+    "state": {
+      "id": 135,
+      "txid": "...",
+      "rawtx": "000022...",
+      "paymentchannelid": "my-id-payment-channel",
+      "updated_at": 1114554332,
+      "created_at": 1114554332
+    }
+  }
+}
+```
+
+### Get Payment Channel History
+
+`GET /api/v1/paymentchannel/:paymentchannelid/history?limit=1000&offset=0`
+
+Return the chronological history of transactions posted to the payment channel.
+
+Response:
+
+```javascript
+{
+  "status": 200,
+  "errors": [],
+  "result": [
+    {
+      "id": 135,
+      "txid": "...",
+      "rawtx": "000022...",
+      "paymentchannelid": "my-id-payment-channel",
+      "updated_at": 1114554332,
+      "created_at": 1114554332,
+    },
+    {
+      "id": 125,
+      "txid": "...",
+      "rawtx": "000022...",
+      "paymentchannelid": "my-id-payment-channel",
+      "updated_at": 1114554332,
+      "created_at": 1114554332,
+    }
+  ]
+}
+```
+
+### Close Payment Channel
+
+`POST /api/v1/paymentchannel/:paymentchannelid/close`
+
+**Note:** The payment channel will automatically be closed if a transaction is broadcast via  `/mapi` or `/tx`
+when one of the inputs being spent collide with any transactions associated with a payment channel.
+
+Client may forcefully close payment channel without broadcasting a transaction. No further additional transactions may be added

```

Comment:
When the channel is closed and the tx is not broadcasted, just save the `rawtx` locally and can broadcast whenever.

Q: Are payment channels pruned from TXQ? Or will all payment channels always exist in the database?

Reply:
Looking at the SQL file I noticed that we don't store in the database whether the channel was closed and the transaction was broadcasted. Would this be valuable to add?

Reply:
>When the channel is closed and the tx is not broadcasted, just save the rawtx locally and can broadcast whenever.

Good point!  The client can fetch the rawtx/latest state of the channel and broadcast, perfect!

>Looking at the SQL file I noticed that we don't store in the database whether the channel was closed and the transaction was broadcasted. Would this be valuable to add?

I haven't really finished the schema. But very good points, I'll add them both in :)